sdks/python/src/honcho/async_client/client.py

```
@@ -37,6 +41,26 @@ class AsyncHoncho(BaseModel):
     )
     _client: AsyncHonchoCore = PrivateAttr()
 
+    @property
+    def core(self) -> AsyncHonchoCore:
+        """
+        Access the underlying honcho_core client. The honcho_core client is the raw Stainless-generated client,
```

Pylint (3.3.7)

[convention] Line too long (111/100)

(C0301)